joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/AnnotationTests.scala

```diff
@@ -0,0 +1,39 @@
+package io.joern.php2cpg.querying
+
+import io.joern.php2cpg.testfixtures.PhpCode2CpgFixture
+import io.shiftleft.semanticcpg.language.*
+
+class AnnotationTests extends PhpCode2CpgFixture {
+  "annotations for nodes" should {
+    "be populated" in {
+      val cpg = code("""
+          |<?php
+          | #[Route("/api")]
+          | class Foo {
+          |   #[Route("/edit", name: "hello")]
+          |   public function bar(#[SomeAttr] $pBar){}
+          | }
+          |>
+          |""".stripMargin)
+
+      inside(cpg.typeDecl("Foo").annotation.l) { case route :: Nil =>
```

Comment:
Could you include the default case with a `fail` call?

Reply:
Same with all tests in this file

Reply:
Done.